conda.recipe/build.sh

```diff
@@ -0,0 +1,6 @@
+SRC_DIR=$RECIPE_DIR/..
+cd $SRC_DIR
+npm install .
+npm run build
+python setup.py install
+python -m nbpresent.install --force --enable
```

Comment:
>@damianavila in your experience, where do most people install extensions (i.e. global/user)? 

Usually in user space...

>Is there an expectation that the global /share location is really shared by every user/app?

In some deployment like JupyterHub, maybe it makes sense to have a system-wide installation...